server/routes/index.ts

```diff
@@ -30,9 +33,33 @@ export default function routes(service: Services): Router {
     const overviewPageService = new OverviewPageService(prisonApi)
     const overviewPageData = await overviewPageService.get(prisonerData)
 
+    const pageConfig: PageConfig = DisplayBanner
+    const pageBodyNjk = './overviewPage.njk'
+
+    res.render('pages/index', {
+      ...mapHeaderData(prisonerData),
+      ...overviewPageData,
+      ...pageConfig,
+      pageBodyNjk,
+    })
+  })
+
+  get('/prisoner/:prisonerNumber/image', async (req, res, next) => {
+    const prisonerSearchClient = new PrisonerSearchClient(res.locals.clientToken)
+    const prisonerData: Prisoner = await prisonerSearchClient.getPrisonerDetails(req.params.prisonerNumber)
+
+    const prisonApi = new PrisonApiRestClient(res.locals.clientToken)
+    const overviewPageService = new OverviewPageService(prisonApi)
+    const overviewPageData = await overviewPageService.get(prisonerData)
```

Comment:
I don't think we need to do all the API calls for the overview page as this just displays the image.
From the template, seems like you just need the prisonerNumber (which is in the request param), photoType and the name, which could come from the first call to `getPrisonerDetails`?

(Maybe something to think about for the future would be if we want to store any session state, like prisoner name, or even just be able to pass objects from one page to the next?)